### PR TITLE
add support for setting quiet mode

### PR DESCRIPTION
**Pull request by swolix**

### CODE REVIEW

File: aioaquarea/data.py [hunk 1]
```diff
@@ -506,3 +506,11 @@ async def set_temperature(
         :param temperature: The temperature to set
         :param zone_id: The zone id to set the temperature for
         """
+
+    @abstractmethod
+    async def set_quiet_mode(
+        self, mode: int
```
Comment (cjaliaga) [inline aioaquarea/data.py:512]: Could you define mode as an Enum `QuietMode`, similar to the way `OperationStatus' or 'OperationMode' are defined with all the posible values?https://github.com/cjaliaga/aioaquarea/blob/617037187915c88e8e7b6ca5ce5d120c85d3d10f/aioaquarea/data.py#L32-L42

**Review by cjaliaga — CHANGES_REQUESTED** [review]
Thanks for your contribution @swolix!

File: aioaquarea/data.py (hunk 1 above)
Comment (swolix) [reply aioaquarea/data.py]: I added the enum, thanks!

File: aioaquarea/core.py [hunk 2]
```diff
@@ -509,6 +509,28 @@ async def _post_device_zone_temperature(
...
+    @auth_required
+    async def post_set_quiet_mode(
+        self, long_id: str, mode: QuietMode
+    ) -> None:
+        """Post quiet mode"""
+        data = {
+            "status": [
+                {
+                    "deviceGuid": long_id,
+                    "quietMode": int(mode),
```
Comment (cjaliaga) [inline aioaquarea/core.py:521]:
```suggestion
                    "quietMode": mode.value
```

**Review by cjaliaga — APPROVED** [review]

**Comment by acamunas** [comment]
it would be beatiful to add support for setting power mode too

**Comment by swolix** [comment]
@acamunas 

> it would be beatiful to add support for setting power mode too

Can you test this; https://github.com/swolix/aioaquarea (I don't use powerful mode)
If you can confirm this works, I'll put up a pull-request

**Comment by acamunas** [comment]
i´m not confident to test it now, its my second week with HA and i'm trying to do my best maybe other experience user can test it

**Pull request merged**